packages/connect-examples/webextension-v3/style.css

```diff
@@ -0,0 +1,24 @@
+body {
+    width: 20rem;
+}
+
+ul {
+    list-style-type: none;
+    padding-inline-start: 0;
+    margin: 1rem 0;
+}
+
+li {
+    padding: 0.25rem;
+}
+li:nth-child(odd) {
+    background: #80808030;
+}
+li:nth-child(even) {
+    background: #ffffff;
+}
+
+h3,
+p {
+    margin: 0;
+}
```

Comment:
is it used? I haven't really noticed any styles :thinking:

Reply:
Removed that https://github.com/trezor/trezor-suite/pull/7699/commits/420dd87e3e07271febe4be829f7b3ecfbf313c37